sjb/config/common/test_cases/origin_release_install_gce.yml

```diff
@@ -70,6 +70,10 @@ extensions:
           exit 0
         fi
         
+        # Allows us to publish artifacts for tags, since we have no tag mechanism
```

Comment:
We either need to fix this upstream or at least open a ticket to Github. LGTM for now.

Reply:
Is there anything existing I can reference?

Reply:
Nm, looks like https://github.com/kubernetes/test-infra/issues/5962